components/governance/org.wso2.carbon.governance.taxonomy.ui/src/main/java/org/wso2/carbon/governance/taxonomy/ui/clients/TaxonomyManagementClient.java

```diff
@@ -39,11 +32,10 @@
  * taxonomy stub
  */
 public class TaxonomyManagementClient {
-    private static final Log log = LogFactory.getLog(TaxonomyManagementClient.class);
     private TaxonomyServicesStub stub;
 
     public TaxonomyManagementClient(String cookie, ServletConfig config, HttpSession session)
-            throws TaxonomyServicesRegistryExceptionException {
+            throws Exception {
```

Comment:
Please review why specific exception types are replaced by the exception super class.